src/CodeFixes/VerifyOverridableMembersFixer.cs

```diff
@@ -0,0 +1,138 @@
+using System.Composition;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Moq.Analyzers;
+
+/// <summary>
+/// Fixes Moq.Verify calls on non-overridable members by making them virtual.
+/// </summary>
+[ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(VerifyOverridableMembersFixer))]
+[Shared]
+public sealed class VerifyOverridableMembersFixer : CodeFixProvider
+{
+    /// <inheritdoc/>
+    public override ImmutableArray<string> FixableDiagnosticIds { get; } = ImmutableArray.Create(DiagnosticIds.VerifyOnlyUsedForOverridableMembers);
+
+    /// <inheritdoc/>
+    public override FixAllProvider GetFixAllProvider() => WellKnownFixAllProviders.BatchFixer;
+
+    /// <inheritdoc/>
+    public override async Task RegisterCodeFixesAsync(CodeFixContext context)
+    {
+        SyntaxNode? root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
+        if (root == null)
+        {
+            return;
+        }
+
+        Diagnostic diagnostic = context.Diagnostics.First();
+        Microsoft.CodeAnalysis.Text.TextSpan diagnosticSpan = diagnostic.Location.SourceSpan;
+
+        SyntaxNode? invocationNode = root.FindNode(diagnosticSpan, getInnermostNodeForTie: true);
+        if (invocationNode is not InvocationExpressionSyntax)
+        {
+            invocationNode = invocationNode.FirstAncestorOrSelf<InvocationExpressionSyntax>();
```

Comment:
This line is uncovered. Write a unit test for this scenario if it is valid; otherwise, refactor so we don't have uncovered lines.